selfdrive/car/__init__.py

```diff
@@ -1,8 +1,36 @@
 # functions common among cars
 from common.numpy_fast import clip
-
+from selfdrive.config import Conversions as CV
 # kg of standard extra cargo to count for drive, gas, etc...
 STD_CARGO_KG = 136.
+
+# FIXME: hardcoding honda civic 2016 touring params so they can be used to
+# scale unknown params for other cars
+class CivicParams:
+  MASS = 2923. * CV.LB_TO_KG + STD_CARGO_KG
+  WHEELBASE = 2.70
+  CENTER_TO_FRONT = WHEELBASE * 0.4
+  CENTER_TO_REAR = WHEELBASE - CENTER_TO_FRONT
+  ROTATIONAL_INERTIA = 2500
+  TIRE_STIFFNESS_FRONT = 192150
+  TIRE_STIFFNESS_REAR = 202500
+
+# TODO: get actual value, for now starting with reasonable value for
+# civic and scaling by mass and wheelbase
+def scale_rot_inertia(mass, wheelbase):
+  return CivicParams.ROTATIONAL_INERTIA * mass * wheelbase ** 2 / (CivicParams.MASS * CivicParams.WHEELBASE ** 2)
+
+# TODO: start from empirically derived lateral slip stiffness for the civic and scale by
+# mass and CG position, so all cars will have approximately similar dyn behaviors
+def scale_tire_stiffness(mass, wheelbase, center_to_front, tire_stiffness_factor):
```

Comment:
tire_stiffness_factor should default to 1, as it's not always necessary (see chrysler)

Reply:
so the gm, chrysler, and hyundai? seemed odd to me at the time

Reply:
well, 
- chrysler we can use the default 1 (close enough to 0.888), 
- hyundai is 1, so also we can use the default.
- for gm we should keep 0.444